src/modules/app/components/Navigator.js

```diff
@@ -62,7 +67,9 @@ export const AppStack = createStackNavigator(
     [WOHNEN_ROUTE]: createNavigationRouteConfig(WohnenExtraContainer, defaultHeader),
     [SPRUNGBRETT_ROUTE]: createNavigationRouteConfig(SprungbrettExtraContainer, defaultHeader),
     [EXTERNAL_EXTRA_ROUTE]: createNavigationRouteConfig(ExternalExtraContainer, defaultHeader),
-    'Events': createNavigationRouteConfig(EventsContainer, defaultHeader),
+    'Events': createNavigationRouteConfig( // $FlowFixMe
```

Comment:
If you have any idea, on how to solve this, don't hesitate to suggest it :D
I cannot see any significant difference of EventsContainer to DashboardContainer or CategoriesContainer why Flow would throw an error here, but not at the other screens... :/

Reply:
I don't have a clue, sorry. But if nobody knows a solution, it's not worth the time to look for a solution.

Reply:
Yeah lets not waste time on this. Maybe just add a comment to the FlowFixMe that we do not know why it happens. Also a task would be good maybe as it is in a central component of the app.